var/spack/repos/builtin/packages/proj/package.py

```diff
@@ -83,12 +84,49 @@ class Proj(AutotoolsPackage):
     )
 
     # https://proj.org/install.html#build-requirements
-    depends_on("pkgconfig@0.9.0:", type="build", when="@6:")
     depends_on("googletest", when="@6:")
     depends_on("sqlite@3.11:", when="@6:")
     depends_on("libtiff@4.0:", when="@7:+tiff")
     depends_on("curl@7.29.0:", when="@7:+curl")
+    depends_on("pkgconfig@0.9.0:", type="build", when="@6: build_system=autotools")
+    depends_on("cmake@2.6.0:", type="build", when="build_system=cmake")
+    # Builds for Windows via CMake available only
+    # after this version - builds before exist
+    # for Windows but are targeted at VS2005
+    conflicts("platform=windows", when="@:5.0.0")
 
+    build_system("autotools", conditional("cmake", when="@5.0.0:"), default="autotools")
+
+
+class Setup(object):
```

Comment:
```suggestion
class Setup:
```

Reply:
We're allowed to use new style class definitions now? Sweet!

Reply:
Yes, Python 3.6 is the lowest version of Python we support now!